flink-libraries/flink-table/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -1900,7 +1900,7 @@ public void unparse(
 		new SqlGroupFunction(SqlKind.TUMBLE, null,
 			OperandTypes.or(OperandTypes.DATETIME_INTERVAL,
 				OperandTypes.DATETIME_INTERVAL_TIME)) {
-			@Override List<SqlGroupFunction> getAuxiliaryFunctions() {
+			@Override public List<SqlGroupFunction> getAuxiliaryFunctions() {
```

Comment:
As I mentioned in the PR description: "We need to get the visibility changes that I made in Calcite in the next Calcite release." Right now it is not possible to add custom SqlGroupFunctions.